src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -30,16 +30,16 @@ public String reservation() {
 
     @ResponseBody
     @GetMapping("/reservations")
-    public List<Reservation> reservations() {
+    public List<ReservationDto> reservations() {
         return reservationService.getAllReservations();
     }
 
     @ResponseBody
     @PostMapping("/reservations")
-    public ResponseEntity<Reservation> addReservation(@RequestBody Reservation reservation) throws URISyntaxException {
-        Reservation responseReservation = reservationService.addReservation(reservation);
-        return ResponseEntity.created(new URI("/reservations/" + responseReservation.id()))
-            .body(responseReservation);
+    public ResponseEntity<ReservationDto> addReservation(@RequestBody ReservationDto reservationDto) throws URISyntaxException {
```

Comment:
dto라는 변수명보다는 request와 같은 네이밍은 어떤가요?
```suggestion
    public ResponseEntity<ReservationDto> addReservation(@RequestBody ReservationDto request) throws URISyntaxException {
```